README.md

```diff
@@ -14,7 +14,7 @@ If you want to help develop `trio-websocket`, clone [the
 repository](https://github.com/hyperiongray/trio-websocket) and run this command
 from the repository root:
 
-    pip install --editable .
+    pip install --editable .[dev]
```

Comment:
personally I never use this and prefer `PYTHONPATH=. ...`

Reply:
The editable flag is nice because `pip` will still install the dependencies for you, which you wouldn't get by just setting `PYTHONPATH`.